packages/ListBox/src/components/Trigger/Trigger.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import PropTypes from "prop-types";
+import RawButton from "@paprika/raw-button";
+import Label from "../Label";
+import handleKeyboardKeys from "../../helpers/handleKeyboardKeys";
+import useListBox from "../../useListBox";
+import applyOnChange, {
+  cleanActionTypes,
+  getSelectedOptionSingle,
+  getSelectedOptionsMulti,
+} from "../../helpers/applyOnChange";
+
+import {
+  ListBoxTriggerStyled,
+  TriggerArrowStyled,
+  RawButtonClearButtonStyled,
+  TriggerActionIconsContainer,
+} from "./Trigger.styles";
+import { getDOMAttributesForListBoxButton } from "../../helpers/DOMAttributes";
+
+const propTypes = {
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+  hasClearButton: PropTypes.bool,
+  onClickClear: PropTypes.func,
+  onFooterClickAccept: PropTypes.func,
+  placeholder: PropTypes.string,
+};
+
+const defaultProps = {
+  children: <React.Fragment />,
```

Comment:
Why not `null`?

Reply:
Was breaking the rendering. that's the reason why I changed it to React.Fragment. But don't remember exactly the reason.